src/monitors/rocketMonitor.cpp

```diff
@@ -81,6 +84,23 @@ PetscErrorCode ablate::monitors::RocketMonitor::OutputRocket(TS ts, PetscInt ste
         PetscReal thrustTotalGlob[3] = {0, 0, 0};
         PetscReal IspGlob[3] = {0, 0, 0};
 
+        // initialize field values to calculate [min max mean] for cell pressure and mach number
+        PetscReal pressureMin[1] = {100000000};
+        PetscReal pressureMax[1] = {0};
+        PetscReal pressureTotal[1] = {0};
+        PetscReal pressureMinGlob[1];
+        PetscReal pressureMaxGlob[1];
+        PetscReal pressureTotalGlob[1];
+        PetscReal machMin[1] = {100000000};
```

Comment:
This do not need to be arrays to send them over mpi.  You can send scalars over mpi by giving the memory address of the scalar as &varName